libs/platform/icon-tab-bar/directives/icon-tab-title.directive.ts

```diff
@@ -0,0 +1,7 @@
+import { Directive } from '@angular/core';
+
+@Directive({
+    selector: '[fdpIconTabBarTitleTemplate], [fdp-icon-tab-bar-title-template]',
+    standalone: true
+})
+export class IconTabTitleDirective {}
```

Comment:
What is the purpose of this directive? As far as I can tell, this adds no benefit, we should just let the developer do this instead:

```
    <fdp-icon-tab-bar-tab>
        <span>User Defined Title</span>
        <div class="tab-container-example">Custom tab content</div>
    </fdp-icon-tab-bar-tab>
```

Reply:
This directive allows customization of the tab title template. It retrieves the tab title's template using contentChild. If a custom template is provided, it will display that; otherwise, the default tab title will be shown.

`readonly titleTemplate = contentChild(IconTabTitleDirective, { read: TemplateRef });`

example

```
<fd-tab-list>
    <fd-tab>
        <ng-template fd-tab-title-template>
         <h3 fd-title>Tabs With Counter Built by Defining Template</h3>
        <fd-text text="Content for tab 3"></fd-text>
    </fd-tab>
</fd-tab-list>
```